File: python23/no_of_elements.py
#Problem 4

def find_element(one):
    if not one:
        return None
    else:
        return len(one)
        

example_list = ['a', 'b', 'c', 'd', 'e']

no_of_elements = find_element(example_list)

print("Number of elements : ", no_of_elements)

File: java23/NoofElements.java
//Problem 4

package java23;

import java.util.List;

public class NoofElements {
    
    public static void main(String[] args) {
        List<String> list = List.of("a", "b", "c", "d", "e");

        if (list.isEmpty()) {
            System.out.println("Empty List");
        } else {
            int noofelements = list.size();
            System.out.println("No of Elements : " + noofelements);
        }
    }
    
}


